app/src/main/res/layout/screen_teleconsult_success.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8"?>
+<org.simple.clinic.teleconsultlog.success.TeleConsultSuccessScreen xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:background="@color/grey4">
+
+  <RelativeLayout
+    android:id="@+id/teleconsultSuccessHeader"
+    android:layout_width="0dp"
+    android:layout_height="wrap_content"
+    android:layout_marginBottom="112dp"
+    android:background="?colorPrimary"
+    android:elevation="@dimen/elevation_toolbar"
+    app:layout_constraintBottom_toTopOf="@+id/imageView"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toTopOf="parent"
+    tools:ignore="UnusedAttribute">
+
+    <ImageButton
+      android:id="@+id/backButton"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_alignParentStart="true"
+      android:layout_alignParentTop="true"
+      android:background="?selectableItemBackgroundBorderless"
+      android:padding="@dimen/spacing_12"
+      android:tint="@color/white"
+      app:srcCompat="@drawable/ic_arrow_back_24dp" />
+
+    <TextView
+      android:id="@+id/patientNameDetailTextView"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_centerVertical="true"
+      android:layout_marginStart="@dimen/spacing_56"
+      android:textAppearance="@style/Clinic.V2.TextAppearance.PatientSummary.PatientName"
+      android:textSize="@dimen/textsize_20"
+      tools:text="Anish Acharya, M, 47" />
+
+  </RelativeLayout>
+
+  <ImageView
+    android:id="@+id/imageView"
+    android:layout_width="130dp"
+    android:layout_height="130dp"
+    app:layout_constraintBottom_toBottomOf="parent"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintHorizontal_bias="0.5"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toTopOf="parent"
+    app:layout_constraintVertical_bias="0.2"
+    app:srcCompat="@drawable/ic_signature_accept"
+    tools:ignore="ContentDescription" />
+
+  <TextView
+    android:id="@+id/textView"
+    style="@style/Clinic.LoginPinLabel"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="@dimen/spacing_16"
+    android:text="@string/screen_teleconsult_success_teleconsult_recorded"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toBottomOf="@+id/imageView" />
+
+  <LinearLayout
```

Comment:
Why do we need this specific `LinearLayout` here if the parent is already a `ConstraintLayout`? We _should_ be able to achieve the same layout with the constraints available to us.

Reply:
Removed `Recorded` text from the layout, but we need a have a parent layout which has white background only for the two material buttons. So keeping `LinearLayout` for that. PTAL?